www/shadowfox-updater/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/SrKomodo/shadowfox-updater 1.7.19 v
+
+categories          www
+platforms           darwin linux
+license             MIT
+maintainers         {@ylluminarious orbitalimpact.com:georgedp} openmaintainer
+description         An auto-updater for ShadowFox (a dark theme for Firefox).
+
+long_description    This is a cross-platform installer/uninstaller/updater for ShadowFox, \
+                    which is a universal dark theme for Firefox. You can visit the main ShadowFox \
+                    website at https://overdodactyl.github.io/ShadowFox
```

Comment:
No need to list the website in the `long_description`. Although you could set this to the `homepage`.

Reply:
@mf2k Ok, I set the main site URL to the `homepage`.

Reply:
@mf2k Actually, on second thought, I left the `homepage` blank. There is some potentially important usage information on the Github page which is not on the main site.